DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/GraphSyncPartGraphSyncer.cs

```diff
@@ -29,5 +32,12 @@ public GraphSyncPartGraphSyncer(IGraphSyncPartIdProperty graphSyncPartIdProperty
 
             return Task.FromResult(Enumerable.Empty<ICommand>());
         }
+
+        public Task<bool> VerifySyncComponent(ContentItem contentItem, INode node, ContentTypePartDefinition contentTypePartDefinition,
+            IEnumerable<IRelationship> relationships, IEnumerable<INode> destNodes)
+        {
+            var uri = node.Properties["uri"];
```

Comment:
This should use the `_graphSyncPartIdProperty` to get the id name (and can be used to extract the value from the graph sync part), similarly to its use in `AddSyncComponents` above.